analyzer/src/funTest/assets/projects/synthetic/gradle-composite-expected-output.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Commit message: Please clean up the footers.
Commit message nit: "Composite builds are a feature..."

Reply:
Marcel is still the author and co-author, is this intended? And shouldn't you be the author of the commit?

```
Co-authored-by: Nicolas Nobelis <nicolas.nobelis@bosch.io>
Co-authored-by: Marcel Bochtler <marcel.bochtler@bosch.io>
Signed-off-by: Nicolas Nobelis <nicolas.nobelis@bosch.io>
Signed-off-by: Marcel Bochtler <marcel.bochtler@bosch.io>
```

Reply:
Fixed.